packages/storybook/src/stories/4-ContractEditor.stories.js

```diff
@@ -52,17 +54,17 @@ This is text. This is *italic* text. This is **bold** text. This is a [link](htt
         clause.parse(template.getMetadata().getSample());
         clause.draft({ wrapVariables: true })
         .then((drafted) => {
-          const clauseMarkdown = `${markdownText}
-\`\`\` <clause src="${templateUrl}" clauseid="123">
+          const clauseMarkdown = `
+\`\`\` <clause src="${templateUrl}" clauseid="${uuid()}">
 ${drafted}
 \`\`\`
-This is some more text. Test moving a clause by dragging it or by using the up and down arrows. This is some more text. Test moving a clause by dragging it or by using the up and down arrows. This is some more text. Test moving a clause by dragging it or by using the up and down arrows. This is some more text. Test moving a clause by dragging it or by using the up and down arrows.
```

Comment:
Don't we still want this so there is text after the clause template in order to drag around?

Reply:
I took it out since now we can just move the clause up and type stuff after it. But if we think it should stay in, I could add it back. Maybe when a clause is added we should always add a few lines after it? I'm gonna try that and see how it feels.

Reply:
Trying to think of where I have code that inserts a blank slate paragraph node when a clause template is added...

Reply:
Yeah I think just in this storybook it makes sense. If we end up adding a knob or whatever that _inserts_ a new template, then that can not insert text after. I think it's actually in the `markdown-transform` that does this in https://github.com/accordproject/markdown-transform/pull/133!

Reply:
The logic here is changed so that it _does_ insert a new template, so I think you are right. I've added text before and after the clause upon insertion. Still not sure if this is the best approach, but I think it is good for now for making development and demoing easier.